Library/Homebrew/cmd/bottle.rb

```diff
@@ -247,6 +239,9 @@ def bottle_formula(f)
         else
           relocatable = !keg_contains(prefix_check, keg, ignores)
           relocatable = !keg_contains(cellar, keg, ignores) && relocatable
+          unless prefix == prefix_check
+            relocatable = !keg_contains_absolute_symlink_starting_with?(prefix, keg) && relocatable
```

Comment:
Can you split this into a few `= true if`s instead? I know that's the case above but it's confusing there too.